script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -295,22 +294,22 @@ And so, waving her hand with a smile, Senpai runs off.
 [sha:f6449ffcdfa7c721ccbc40468836b6863dbb4283]{
 -- Page 66, Offset 1323.
 -- ばびゅーん、なんて擬音が似合いそうな、それはそれは軽快かつ高速の駆け足だった。
-It's such a light and fast run that an onomatopoeic sound like %{i}boing%{/i} would be appropriate.
+It's such a light and fast run that an onomatopoeic %{i}boing%{/i} would be appropriate.
 }
 [sha:0ae0330ac391e82876f7a84b2eeb80ce2a586e11]{
 -- Page 66, Offset 1324.
 -- 思わず負けじと走り出したくなったが、こっちには急ぐ理由がない。
-I instinctively want to break into a run as well so as not to be outdone, but I have no particular reason to hurry.
+I almost want to break into a run as well so as not to be outdone, but I have no particular reason to hurry.
```

Comment:
I almost want to break into a run as well, to not be outdone, but I have no particular reason to hurry.

Added a comma and reduced sentence complexity.

Reply:
Not sure I agree, `so as` form feels more fluent.

Reply:
Sure, I'm not precious about it. I think it does need the comma, though.